### PR TITLE
Align naming in Thermostat cluster XML with the spec.

### PR DESCRIPTION
**Pull request by bzbarsky-apple**
REVIEW NOTE: Only the first commit contains manual changes. The rest is code-generated.

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml [hunk 1]
```diff
@@ -213,15 +213,15 @@ limitations under the License.
     <field name="Heating" mask="0x1"/>
     <field name="Cooling" mask="0x2"/>
     <field name="Occupancy" mask="0x4"/>
-    <field name="Schedule" mask="0x8"/>
+    <field name="ScheduleConfiguration" mask="0x8"/>
```
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml:216]: Spec technically has a space in the feature name. Not sure if we care. Same for `Auto Mode`

**Review by andy31415 — COMMENTED** [review]
@bzbarsky-apple should we have a `GetRelayStatusLog` command somewhere in commants? I seem to have it in my spec but I do not see it in the xml definition.

**Review by andy31415 — COMMENTED** [review]
@bzbarsky-apple AlarmMask / map8 / 0x001d does not seem to be in the XML

**Review by andy31415 — COMMENTED** [review]
@bzbarsky-apple SetpointChangeSourceTimeStamp is marked epoch_s here, however in the spec it is UTC.

File: src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml (hunk 1 above)
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml:216]: We do care, but we want to fix that in the spec.

**Comment by bzbarsky-apple** [comment]
> @bzbarsky-apple should we have a GetRelayStatusLog command somewhere in commants? I seem to have it in my spec but I do not see it in the xml definition.

It's in the spec, but it's Zigbee-only in the spec, so should not be here, no.  See "9.2.5. Element Requirements" in the device library spec.

> @bzbarsky-apple AlarmMask / map8 / 0x001d does not seem to be in the XML

Zigbee-only.

> @bzbarsky-apple SetpointChangeSourceTimeStamp is marked epoch_s here, however in the spec it is UTC.

utc is a deprecated alias for epoch_s in the spec.

**Review by andy31415 — APPROVED** [review]

**Pull request merged**